_content/articles/gleanings-from-the-mahavastu_marciniak-katarzyna.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Gleanings from the Mahāvastu"
+authors:
+  - "Katarzyna Marciniak"
+external_url: "https://www.academia.edu/59240112/Gleanings_from_the_Mah%C4%81vastu"
+drive_links:
+  - "https://drive.google.com/file/d/16H6Bd-7FZi5h-aVJv2Ac1ofUwg28-w7S/view?usp=sharing"
```

Comment:
Actually, here's a better PDF:

https://drive.google.com/file/d/12YajvJfi3ZyK17g5aJArEQD_7Fk7vu65/view?usp=drivesdk

Sorry about that! 😊